robot/Cumulus/tests/browser/relationships/create_relationships.robot

```diff
@@ -1,48 +1,69 @@
-    Click Modal Button        Save
-    ${expected result1}        Catenate    &{contact1}[FirstName] &{contact1}[LastName]    is    &{contact2}[FirstName] &{contact2}[LastName]'s    Parent
-    Log to Console    ${expected result1}
-    ${id}    ${status1}    Check Status    &{contact1}[FirstName] &{contact1}[LastName]
-    Should Be Equal As Strings    ${status1}    ${expected result1}
-    Click Link    link=Show more actions
-    Click Link    link=Relationships Viewer
-    Sleep    5 
+    [Documentation]                      Create 2 contacts with API which inturn creates 2 household accounts.
+    ...                                  Open contact2 record and create a new relationship with contact 1.
+    ...                                  Verify the parent-child relationship is correctly established.
+
+    [tags]                              W-037650                                                    feature:Relationships
+
+    Go To Page                          Details                                                     Contact                                               object_id=&{contact2}[Id]
+    Select Tab                          Related
+    Wait Until Loading Is Complete
+    Click Related List Button           Relationships                                               New
+    Wait For Modal                      New                                                         Relationship
+
+    Populate Modal Form
+    ...                                 Related Contact=&{contact1}[FirstName] &{contact1}[LastName]
+    ...                                 Type=Parent
+    Click Modal Button                  Save
+    Current Page Should Be              Details                                                     Contact
+    Validate Relation Status Message    &{contact1}[FirstName] &{contact1}[LastName]
+    ...                                 &{contact2}[FirstName] &{contact2}[LastName]
+    ...                                 Parent
+
+    Click Link                          link=Show more actions
+    Click Link                          link=Relationships Viewer
+    Wait Until Loading Is Complete
     Capture Page Screenshot
-    Go To Record Home    &{contact1}[Id]
-    Select Tab  Related
-    Load Related List    Relationships
-    ${expected result2}        Catenate    &{contact2}[FirstName] &{contact2}[LastName]    is    &{contact1}[FirstName] &{contact1}[LastName]'s    Child
-    Log to Console    ${expected result2}
-    ${id}    ${status2}    Check Status    &{contact2}[FirstName] &{contact2}[LastName]
-    Should Be Equal As Strings    ${status2}    ${expected result2}
-    Click Related Item Link      Relationships    &{contact2}[FirstName] &{contact2}[LastName]
-    Save Current Record ID For Deletion    npe4__Relationship__c    
+    Go To Page                          Details                                                     Contact                                               object_id=&{contact1}[Id]
```

Comment:
Please add a comment here indicating steps for verification of reciprocal relationship